### DISCUSSION TITLE
Orca w/o Webview integration

### DISCUSSION THREAD

**Discussion by Caldoric** [OrcaSlicer/OrcaSlicer#6181] [Ideas]
Speaking as someone who's gone to great length to thoroughly remove all traces of the bloat/spyware commonly known as "Microsoft Edge" from my computer (I exclusively use Firefox), it's somewhat disheartening to find, after first download and install, that this program "requires Webview 2 for certain features", apparently. As such, I would like to heartily request that future Windows versions of this program be made without integration of WV2, so everyone has access to all features. And before anyone else says it, I'm aware that - supposedly - one is meant to be able to install WV2 by itself, separately from Edge, but it doesn't work: the installers claim it's already installed, but programs looking to use it insist it doesn't exist. Anyways, if there's some kind of internet access needed for various features, why not just add something that just hooks into whatever the user's default browser is, instead of going with Edge?

**Comment by kamp2000** [comment]
I don't like edge...

**Comment by chemicallyfried** [comment]
have figured out a fix? its actually cancer that it requires Edge.

**Comment by nekomimimode-hex** [comment]
in win10, WebView2 Runtime 151.0.4129.59 will makes the start page hole blank
